app/models/persistent_volume.rb

```diff
@@ -3,4 +3,7 @@ class PersistentVolume < ContainerVolume
   include NewWithTypeStiMixin
   serialize :capacity, Hash
   delegate :name, :to => :parent, :prefix => true
+  has_many :container_volumes, :through => :persistent_volume_claim
```

Comment:
@nimrodshn Can you add a test for this relation too (name, count)? I have a feeling that we might have a problem here when a PV is bounded to a PVC then it will point itself.